### ISSUE TITLE
question - Pytorch GPU inference examples?

### ISSUE THREAD

**Issue by A-Jacobson** [ucbrise/clipper#615]
Are there any examples or documentation related to using clipper to serve models that perform inference on the gpu?

**Comment by rkooo567** [comment]
Hi @A-Jacobson! I think there is no tutorial for GPU inference right now. I will try to find if anyone is working on it.

**Comment by RehanSD** [comment]
Hi @A-Jacobson! Adding support for inference Omni CUDA enabled GPU’s is a feature we are currently implementing! I’ll be sure to leave you a note when we get it up!

**Comment by RehanSD** [comment]
Hi @A-Jacobson! Just submitted a [PR](https://github.com/ucbrise/clipper/pull/629) for CUDA support for PyTorch! Go ahead and take a look, and follow up if you have any more questions!

**Comment by A-Jacobson** [comment]
nice! I'll be trying it out tomorrow then.

**Comment by A-Jacobson** [comment]
@RehanSD I think your code pointed me in the right direction but I'm running on Kubernetes and it appears you only altered the docker container manager file. Is there a nvidia runtime equivalent for the kubernetes container manager?

**Comment by simon-mo** [comment]
@A-Jacobson For Kubernetes we need to modify the configuration yaml  according to the [doc](https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/#v1-8-onwards)

So we need to add these lines 

```
      resources:
        limits:
          nvidia.com/gpu: 1 # requesting 1 GPU
```
to the bottom of the config file
https://github.com/ucbrise/clipper/blob/c3feae088552ffecb5aa8fb91895d4f4055ba43b/clipper_admin/clipper_admin/kubernetes/model-container-template.yaml#L24-L45

I believe @RehanSD is working on a followup PR for this.

**Comment by RehanSD** [comment]
Addressed in #669
